dev/build/windows/makecoq_mingw.sh

```diff
@@ -1439,10 +1439,14 @@ function make_addon_equations {
 function make_addons {
   if [ -n "${GITLAB_CI}" ]; then
     export CI_BRANCH="$CI_COMMIT_REF_NAME"
-    if [[ ${CI_BRANCH#pr-} =~ ^[0-9]*$ ]]
-    then
+    if [[ ${CI_BRANCH#pr-} =~ ^[0-9]*$ ]]; then
       export CI_PULL_REQUEST="${CI_BRANCH#pr-}"
+    else
+      export CI_PULL_REQUEST=""
     fi
+  else # assume local
+    export CI_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+    export CI_PULL_REQUEST=""
```

Comment:
You are aware that this script usually doesn't run inside a git repository? I don't understand what git is supposed to do here.

There are two typical ways how this script gets Coq:
1.) As a zip from github
2.) From a local folder (might be git or not)
In either case the coq code ends up in /build/coq-xyz, which never has any git stuff in it.
The makecoq_mingw.sh script usually has /build as working directory and enters the component subdirectories with build_prep.

Maybe you can explain what exactly you want to achieve in which build environment and then we discuss how to achieve this.

Please don't forget the initial purpose of these scripts: setup a clean environment and do everything in a clean room way.